rector.php

```diff
@@ -10,6 +10,8 @@
 
 return static function (ContainerConfigurator $containerConfigurator): void
 {
+    $containerConfigurator->import(SetList::PHP_80);
```

Comment:
Just a tip for esaier maintenance of PHP version. There is now "up to" one liner: https://getrector.org/blog/new-in-rector-012-the-latest-php-in-a-single-import